src/app/models/user.ts

```diff
@@ -0,0 +1,41 @@
+export interface User {
```

Comment:
perchè hai 2 file con interfacce e enums uguali?

Reply:
In realtà stavo facendo l'esercizio con Fede che mi aveva detto di aver riscontrato errori a usare l'interface di User in questo esercizio, così abbiamo creato una interfaccia con un nome diverso anche se con le stesse proprietà